iontestdata/good/equivs/nonIVMNoOps.ion

```diff
@@ -0,0 +1,8 @@
+// Unannotated top-level symbols with text equivalent to the IVM but without IVM semantics are no-ops.
+
+embedded_documents::[
+    "",
+    "'$ion_1_0'",
+    "$2",
+    "$ion_symbol_table::{symbols:['''$ion_1_0''']} $10"
```

Comment:
Maybe add some cases that make sure the reset doesn't happen, e.g. add another symbol and do $11 after $10.